demisto_sdk/commands/validate/validators/RM_validators/RM109_is_readme_exists.py

```diff
@@ -0,0 +1,45 @@
+
+from __future__ import annotations
+
+from typing import Iterable, List, Union
+
+from demisto_sdk.commands.common.constants import GitStatuses
+from demisto_sdk.commands.common.tools import get_pack_name
+from demisto_sdk.commands.content_graph.objects.integration import Integration
+from demisto_sdk.commands.content_graph.objects.pack import Pack
+from demisto_sdk.commands.content_graph.objects.playbook import Playbook
+from demisto_sdk.commands.content_graph.parsers.related_files import RelatedFileType
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Union[Integration, Pack, Playbook]
```

Comment:
```suggestion
ContentTypes = Union[Integration, Script, Playbook]
```
We have a separate validation for packs.
Scripts can also have readmes so let's check that as well (same behaviour as the old validation)